LongWarOfTheChosen/Config/XComGameData.ini

```diff
@@ -1295,20 +1295,24 @@ MedicalModeHealRateScalar[3]=0.25
 -SitReps="InformationWarSitRep"
 -SitReps="PrivateChannelsSitRep"
 -SitReps="MentalFortitude"
--SitReps="DarkEventBarrierSitRep"
--SitReps="DarkEventDarkTowerSitRep"
+; -SitReps="DarkEventBarrierSitRep"
+; -SitReps="DarkEventDarkTowerSitRep"
 ;-SitReps="ChallengeNoReinforcements" DON'T Disable this as it only applies to TQL and skirmish mode and swarms you with redscreens. Really annoying
 
-
-+SitReps=LowVisibility
```

Comment:
While we're at it, can we remove location scout? That sitrep is incredibly cheesy

Reply:
Yeah. I was wondering about a sit rep that just gives all your soldiers Tracking (say it's something like motion tracking from somewhere). Might be a suitable replacement.

Reply:
I'll remove it once we have a replacement. It makes a nice change of pace for the moment.